packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -36,6 +36,41 @@ export type ElementProps = {
     dragging: boolean;
 };
 
+// Using a class element for a change because `componentDidCatch` only works with class elements ;/
+class ErrorBoundary extends React.Component {
```

Comment:
I would ask you to separate this into another file. The convention is to have 1 component per file. Just create a file right next to `Element` - `ErrorBoundary.tsx` and import that.